lib/jsdom/living/progress-event.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+
+module.exports = function (core) {
+  class ProgressEvent extends core.Event {
+    constructor(eventType, eventInit) {
+      super(eventType);
+      if (eventInit) {
+        this.lengthComputable = eventInit.lengthComputable || 0;
```

Comment:
Do `Number(eventInit.lengthComputable) || 0` here and below to get correct type coercion.